### ISSUE TITLE
[Android] Generating Android project for AtomSampleViewer fails

### ISSUE THREAD

**Issue by LB-KacperKapusta** [o3de/o3de-atom-sampleviewer#408]
**Describe the bug**
Running the **generate_android_project.py** script fails for AtomSampleViewer set as active project with the following error:
`Missing key from android project settings for project at C:\o3de-atom-sampleviewer:''android_settings''`

Please note that the same script generates the Android project for AutomatedTesting when it is set as active without any issues.

Please refer to the attached logs for more information.

**Prerequisites**
- AtomSampleViewer project is built for Windows.
- Assets for the Android platform are enabled and have been processed.
- All the required environment variables are set as described in the instructions: https://o3de.org/docs/user-guide/platforms/android/generating_android_project_windows/

In this test the following settings have been used:

```
SET O3DE_ENGINE_PATH=C:\o3de
SET O3DE_PROJECT_PATH=C:\o3de-atom-sampleviewer
SET O3DE_BUILD_ROOT=C:\o3de-build
```
```
SET O3DE_ANDROID_SIGNCONFIG_FILE=%O3DE_BUILD_ROOT%\o3de.keystore
SET O3DE_ANDROID_SIGNCONFIG_STORE_PASSWORD=o3de_android
SET O3DE_ANDROID_SIGNCONFIG_KEY_ALIAS=o3de_key
SET O3DE_ANDROID_SIGNCONFIG_KEY_PASSWORD=o3de_android
SET O3DE_ANDROID_SIGNCONFIG_KEY_SIZE=2048
SET O3DE_ANDROID_SIGNCONFIG_VALIDITY_DAYS=10000
SET O3DE_ANDROID_DN="cn=atom-sample-viewer, ou=o3de, o=LF, c=US"
```
```
set O3DE_ANDROID_SDK_PATH=C:\Android\Sdk
set O3DE_ANDROID_NDK_VERSION=22.1.7171670
set O3DE_ANDROID_SDK_API_LEVEL=30
set O3DE_ANDROID_ASSET_MODE=LOOSE
set O3DE_ANDROID_DEPLOY_TYPE=BOTH
```

**Steps to reproduce**
1. Generate KeyStore to allow APK signing:
`keytool -genkey -keystore %O3DE_ANDROID_SIGNCONFIG_FILE% -storepass %O3DE_ANDROID_SIGNCONFIG_STORE_PASSWORD% -alias %O3DE_ANDROID_SIGNCONFIG_KEY_ALIAS% -keypass %O3DE_ANDROID_SIGNCONFIG_KEY_PASSWORD% -keyalg RSA -keysize %O3DE_ANDROID_SIGNCONFIG_KEY_SIZE% -validity %O3DE_ANDROID_SIGNCONFIG_VALIDITY_DAYS% -dname %O3DE_ANDROID_DN%`
2. Use Python to run generate_android_project.py:
`%O3DE_ENGINE_PATH%\python\python.cmd %O3DE_ENGINE_PATH%\cmake\Tools\Platform\Android\generate_android_project.py --engine-root %O3DE_ENGINE_PATH% --project-path %O3DE_PROJECT_PATH% --build-dir %O3DE_BUILD_ROOT%\android --third-party-path %USERPROFILE%\.o3de\3rdParty --android-sdk-path %O3DE_ANDROID_SDK_PATH% --android-ndk-version %O3DE_ANDROID_NDK_VERSION% --android-sdk-platform %O3DE_ANDROID_SDK_API_LEVEL% --include-apk-assets --asset-mode %O3DE_ANDROID_ASSET_MODE% --signconfig-store-file %O3DE_ANDROID_SIGNCONFIG_FILE% --signconfig-store-password %O3DE_ANDROID_SIGNCONFIG_STORE_PASSWORD% --signconfig-key-alias %O3DE_ANDROID_SIGNCONFIG_KEY_ALIAS% --signconfig-key-password %O3DE_ANDROID_SIGNCONFIG_KEY_PASSWORD%`

**Expected behavior**
Generating the Android project finishes successfully.

**Actual behavior**
Generating the Android project for AtomSampleViewer fails.

**Assets required**
https://github.com/o3de/o3de-atom-sampleviewer

**Found in Branch**
ASV: Stabilization/2205([3efc7e3](https://github.com/o3de/o3de-atom-sampleviewer/commit/3efc7e3ae3d7441257ab3e68ccc457ff6b0f8491))
O3DE: Stabilization/2205([3e47df2](https://github.com/o3de/o3de/commit/3e47df2e0fea9f57d207e6c739938bd1c4019c98))

**Desktop/Device:**
 - Device: PC
 - OS: Windows
 - Version: 10
 - CPU: Intel Core i5-9600K
 - GPU: NVIDIA GeForce RTX 2070
 - Memory: 16GB

**Additional context**
[AndroidLog.txt](https://github.com/o3de/o3de-atom-sampleviewer/files/8464325/AndroidLog.txt)

**Comment by moraaar** [comment]
I was able to build AtomSampleViewer on Android with the latest code. This is most likely just an old issue.

@LB-KacperKapusta I'm going to close this issue for now, but feel free to reopen if you're still having issues.

**Comment by LB-KacperSzczepanski** [comment]
The issue verified as Fixed in Stabilization/2210 (67014ed).
